lrathoni/Lullaby-for-a-fish
# Unity project

### What

* Changed name of the Unity project in order to avoid spaces.
* Added placeholders in subfolders of Assets/ for them to be added to the repository.
* Fixed LFS inclusion of the Unity .gitattributes
* Added some patterns to the Unity .gitattributes corresponding to music formats that we'll use (.MPTM, .ITI ..)

### How to test

* Checkout this branch, `unityProject`.
* Edit: There should still be errors regarding .gittatributes, but they should be fixed once the branch is merged.
* Check that the .MPTM file in Assets/Audio/Music is treated as binary and managed by git LFS.
